boards/nxp/imx93_evk/doc/index.rst

```diff
@@ -126,6 +146,45 @@ display the following ram console output:
     thread_a: Hello World from cpu 0 on mimx93_evk_a55!
     thread_b: Hello World from cpu 0 on mimx93_evk_a55!
 
+Programming and Debugging (M33)
+*******************************
+
+Copy the compiled ``zephyr.bin`` to the first FAT partition of the SD card and
+plug the SD card into the board. Power it up and stop the u-boot execution at
+prompt.
+
+Use U-Boot to load and kick zephyr.bin to Cortex-M33 Core:
+
+.. code-block:: console
+
+    load mmc 1:1 0x80000000 zephyr.bin;cp.b 0x80000000 0x201e0000 0x30000;bootaux 0x1ffe0000 0
+
+Use this configuration to run basic Zephyr applications and kernel tests,
+for example, with the :zephyr:code-sample:`synchronization` sample:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/synchronization
+   :host-os: unix
+   :board: imx93_evk/mimx9352/m33
+   :goals: run
+
+This will build an image with the synchronization sample app, boot it and
+display the following ram console output:
```

Comment:
it is uart console but not ram console, right? so remove "ram",  same error is for A55 Core, please fix it also.